bson/src/test/unit/org/bson/BsonWriterSpecification.groovy

```diff
@@ -17,14 +17,13 @@
 package org.bson
 
 import org.bson.io.BasicOutputBuffer
-import org.junit.Test
 import spock.lang.Specification
 
 class BsonWriterSpecification extends Specification {
 
-    def shouldThrowExceptionForBooleanWhenWritingBeforeStartingDocument() {
+    def 'shouldThrowExceptionForBooleanWhenWritingBeforeStartingDocument'() {
```

Comment:
Is quoting required now in Spock?

Reply:
I can't recall if this broke it or I was just normalizing the test. I think it may only be required if there are spaces in the name of the test, which is probably the normal naming approach for spec style tests.

Reply:
That's what I recall as well.  Feel free to remove the quotes, but I'll resolve this.